server/routing/game.go

```diff
@@ -13,27 +18,54 @@ const (
 
 // GameRoutes contains information about routes specific to game interaction
 type GameRoutes struct {
-	Service services.GameService
+	Services *services.MasterList
 }
 
 // AddRoutes will add all game routes to the given router
 func (gr *GameRoutes) AddRoutes(r *mux.Router) {
-	r.HandleFunc(gameRoute+"/{gameName}/tick/{tick}/player/{playerName}/cards", gr.submitCardsHandler).Methods("PUT")
-	r.HandleFunc(gameRoute+"/{gameName}/tick", gr.getCurrentTickHandler).Methods("GET")
-	r.HandleFunc(gameRoute+"/{gameName}/tick/{tick}/player/{playerName}", gr.getGameStateHandler).Methods("GET")
+	r.HandleFunc(gameRoute+"/{gameID}/turn/{turn}/player/{playerName}", gr.submitCardsHandler).Methods("PUT")
+	r.HandleFunc(gameRoute+"/{gameID}/turn", gr.getCurrentTurnHandler).Methods("GET")
+	r.HandleFunc(gameRoute+"/{gameID}/turn/{turn}/player/{playerName}", gr.getGameStateHandler).Methods("GET")
 }
 
-// submitCardsHandler handles a player request to submit cards for the given tick
+// submitCardsHandler handles a player request to submit cards for the given turn
 func (gr *GameRoutes) submitCardsHandler(w http.ResponseWriter, r *http.Request) {
-	panic("Not yet implemented")
+	w.Write([]byte("Success, though nothing has been recorded"))
```

Comment:
It probably wouldn't get called directly in this method, but somewhere in the stack here.  But check out rules.input_rules.go.ApplyInputRules to validate the inputs that are received by a player against the current state.